subo/release/check_install.go

```diff
@@ -0,0 +1,20 @@
+package release
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/hashicorp/go-version"
+)
+
+func InstallInfo(v *version.Version) string {
```

Comment:
Since we agreed to point the user to https://github.com/suborbital/subo#installing, we can remove the version reference 
```suggestion
func InstallInfo() string {
```